src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+class Deadline extends Task {
+
+    protected LocalDate by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = LocalDate.parse(by);
+    }
+    @Override
+    LocalDate getTime(){
+        return this.by;
+    }
```

Comment:
According to coding conventions, would it be better to write out the full definition of the method here?